Extensions/ExampleJsExtension/JsExtension.js

```diff
@@ -32,6 +32,9 @@ module.exports = {
       'Florian Rival',
       'MIT'
     );
+    extension
```

Comment:
can you remind me why we need this DummyExtension?

Reply:
As an example for anyone wanting to try a new extension. A sort of living documentation. Also can be useful to quickly check that an extension can do something or as a sandbox.